### ISSUE TITLE
resource/aws_cloudwatch_event_target: Intermittent acceptance test failures with active tasks

### ISSUE THREAD

**Issue by gdavison** [hashicorp/terraform-provider-aws#18601]
Various acceptance tests for `aws_cloudwatch_event_target` fail intermittently with the error

```
Error: Error deleting ECS cluster: ClusterContainsTasksException: The Cluster cannot be deleted while Tasks are active.
```

### Affected Resource

* aws_cloudwatch_event_target

**Comment by ewbankkit** [comment]
I just hit this while retesting #18491. I think it's because there is no explicit dependency between the cluster and the task and Terraform attempts to delete the cluster before or in parallel with deleting the task.

**Comment by ewbankkit** [comment]
This is a duplicate of #15652 and #15668.
